### PR TITLE
Fix 'from' pattern option description

### PR DESCRIPTION
**Pull request by gribnoysup**
This PR contains a:

- [ ] **bugfix**
- [ ] new **feature**
- [ ] **code refactor**
- [ ] **test update** 
- [x] **typo fix**
- [ ] **metadata update**
- [x] **documentation update**

### Motivation / Use-Case

As someone who copy-pastes examples before thoroughly reading the description I spent quite a while figuring out why my code doesn't work 😜

This PR fixes `from` pattern option example in the documentation

### Breaking Changes

None

### Additional Info

✌️

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,8 +100,8 @@ module.exports = {
       '/absolute/path/to/dir',
       '**/*',
       {
-        from: '**/*',
-        globOptions: {
```
Comment (anikethsaha) [inline README.md]: dont remove `globOptions`, this plugin supports this options 
https://github.com/webpack-contrib/copy-webpack-plugin/blob/master/src/preProcessPattern.js#L36

File: README.md [hunk 2]
```diff
@@ -100,8 +100,8 @@ module.exports = {
...
+        from: {
+          glob: '**/*',
```
Comment (anikethsaha) [inline README.md]: Keep both,

`globOptions` and `from.glob` both are supported

Comment (gribnoysup) [reply README.md]: Hey @anikethsaha thanks for the review!

I think just adding both options to one example makes it confusing as one of them will take precedence over the other and this example in particular is describing the `from` pattern option, not `globOptions`. Do you think it makes sense to add `globOptions` as a separate entry to the doc as all the other options on the pattern?

I also noticed that the typedefs don't support `globOptions` but if I get it right this should be fixed in [DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped) repo, right?

Comment (anikethsaha) [reply README.md]: Its actually a separate option so It should be documented as a separate options/entry.
Yes about the type it should be changed but I am not sure whether it been released or not?

cc @evilebottnawi

Comment (evilebottnawi) [reply README.md]: Yes, globOptions and from.glob both are supported, feel free to fix it in `DefinitelyTyped`, in next major release we will remove `from.glob` in favor `globOptions` so we can rewrite documentation using `globOptions` (less misleading)

Comment (gribnoysup) [reply README.md]: Thanks for the feedback y'all, I'll update the PR with new section on `globOptions` then 👍

**Pull request merged**